### ISSUE TITLE
one youtube URL download error

### ISSUE THREAD

**Issue by xczheng** [rg3/youtube-dl#14733]
Please see this one below:

youtube-dl  -v " https://www.youtube.com/watch?v=FvFRDL0Aca4"
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', ' https://www.youtube.com/watch?v=FvFRDL0Aca4']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.11.06
[debug] Python version 3.6.0 - Linux-3.2.61-grsec-modsign-x86_64-with-debian-jessie-sid
[debug] exe versions: avconv 9.20-6, avprobe 9.20-6, ffmpeg N-87921-gdf45ea45df-static, ffprobe N-87921-gdf45ea45df-static
[debug] Proxy map: {}
[youtube] FvFRDL0Aca4: Downloading webpage
[youtube] FvFRDL0Aca4: Downloading video info webpage
[youtube] FvFRDL0Aca4: Extracting video information
[youtube] FvFRDL0Aca4: Downloading MPD manifest
WARNING: Your copy of avconv is outdated and unable to properly mux separate video and audio files, youtube-dl will download single file media. Update avconv to version 10-0 or newer to fix this.
[debug] Default format spec: best/bestvideo+bestaudio
[debug] Invoking downloader on 'https://r8---sn-p5qlsnlr.googlevideo.com/videoplayback?key=yt6&ip=2607%3Af298%3A5%3A6000%3A%3Ac0b%3Acce1&lmt=1510089918251993&ms=au&source=youtube&mv=u&dur=3130.142&id=o-AH0g1NjYzPe6L1pRl9-08ndUucMV_ssFFuL6qQ0g_A2l&expire=1510530401&mm=31&signature=79C1A150A58357C64258E85B64FF1DF497210D3E.15758A7B1C6540147DD36DCDDBA3708A45B2B2C8&mn=sn-p5qlsnlr&gcr=us&mime=video%2Fmp4&itag=22&requiressl=yes&mt=1510507957&ratebypass=yes&pl=48&ei=AYkIWs37FNms8wTf1qXACQ&ipbits=0&sparams=dur%2Cei%2Cgcr%2Cid%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cexpire'
[download] Resuming download at byte 1399622

ERROR: Did not get any data blocks
  File "/home/colinz/bin/youtube-dl", line 11, in <module>
    load_entry_point('youtube-dl==2017.11.6', 'console_scripts', 'youtube-dl')()
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/__init__.py", line 465, in main
    _real_main(argv)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/__init__.py", line 455, in _real_main
    retcode = ydl.download(all_urls)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 1985, in download
    url, force_generic_extractor=self.params.get('force_generic_extractor', False))
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 795, in extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 849, in process_ie_result
    return self.process_video_result(ie_result, download=download)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 1619, in process_video_result
    self.process_info(new_info)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 1892, in process_info
    success = dl(filename, info_dict)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 1831, in dl
    return fd.download(name, info)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/downloader/common.py", line 361, in download
    return self.real_download(filename, info_dict)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/downloader/http.py", line 287, in real_download
    download()
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/downloader/http.py", line 257, in download
    self.report_error('Did not get any data blocks')
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/downloader/common.py", line 163, in report_error
    self.ydl.report_error(*args, **kargs)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 612, in report_error
    self.trouble(error_message, tb)
  File "/home/colinz/lib/python3.6/site-packages/youtube_dl/YoutubeDL.py", line 574, in trouble
    tb_data = traceback.format_list(traceback.extract_stack())

**Comment by al1fr** [comment]
No problem to download this video with the last windows version of youtube-dl

**Comment by xczheng** [comment]
The one that has the problem is on Linux.

**Comment by Hrxn** [comment]
> WARNING: Your copy of avconv is outdated and unable to properly mux separate video and audio files, youtube-dl will download single file media. Update avconv to version 10-0 or newer to fix this.

Maybe try to fix this and then try again?

**Comment by xczheng** [comment]
No, the avconv version warning is not the issue.

**Comment by dstftw** [comment]
Most likely a connectivity issue. Try downloading a different format with `-f`.

**Comment by raszpl** [comment]
CAN reproduce
```
D:\_learning>youtube-dl -f 22 -g  https://www.youtube.com/watch?v=FvFRDL0Aca4
https://r5---sn-f5f7lne6.googlevideo.com/videoplayback?sparams=dur%2Cei%2Cgcr%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cexpire&signature=9A40973E10581CBAFEC407C289DCC9D2ADD43AF2.A729230062EA5A24F0CA1565FD3A07E9837D16B6&itag=22&c=WEB&id=o-AA-YvKNVrqlsUju19HHHyW_YI6osduGtVnZoB0QZj0eQ&mn=sn-f5f7lne6%2Csn-4g5ednly&mm=31%2C26&ip=&ms=au%2Conr&ei=ZQLMWpTxB7ym7ATgw4-wCw&mv=m&pl=13&ipbits=0&dur=3130.142&fvip=5&source=youtube&lmt=1510089918251993&ratebypass=yes&expire=1523340997&key=yt6&mime=video%2Fmp4&gcr=pl&mt=1523319283&requiressl=yes&initcwndbps=1318750

D:\_learning>wget.exe "https://r5---sn-f5f7lne6.googlevideo.com/videoplayback?sparams=dur%2Cei%2Cgcr%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cexpire&signature=9A40973E10581CBAFEC407C289DCC9D2ADD43AF2.A729230062EA5A24F0CA1565FD3A07E9837D16B6&itag=22&c=WEB&id=o-AA-YvKNVrqlsUju19HHHyW_YI6osduGtVnZoB0QZj0eQ&mn=sn-f5f7lne6%2Csn-4g5ednly&mm=31%2C26&ip=&ms=au%2Conr&ei=ZQLMWpTxB7ym7ATgw4-wCw&mv=m&pl=13&ipbits=0&dur=3130.142&fvip=5&source=youtube&lmt=1510089918251993&ratebypass=yes&expire=1523340997&key=yt6&mime=video%2Fmp4&gcr=pl&mt=1523319283&requiressl=yes&initcwndbps=1318750"
--2018-04-10 03:17:08--  https://r5---sn-f5f7lne6.googlevideo.com/videoplayback?sparams=dur%2Cei%2Cgcr%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cexpire&signature=9A40973E10581CBAFEC407C289DCC9D2ADD43AF2.A729230062EA5A24F0CA1565FD3A07E9837D16B6&itag=22&c=WEB&id=o-AA-YvKNVrqlsUju19HHHyW_YI6osduGtVnZoB0QZj0eQ&mn=sn-f5f7lne6%2Csn-4g5ednly&mm=31%2C26&ip=&ms=au%2Conr&ei=ZQLMWpTxB7ym7ATgw4-wCw&mv=m&pl=13&ipbits=0&dur=3130.142&fvip=5&source=youtube&lmt=1510089918251993&ratebypass=yes&expire=1523340997&key=yt6&mime=video%2Fmp4&gcr=pl&mt=1523319283&requiressl=yes&initcwndbps=1318750
Resolving r5---sn-f5f7lne6.googlevideo.com (r5---sn-f5f7lne6.googlevideo.com)... 172.217.133.74
Connecting to r5---sn-f5f7lne6.googlevideo.com (r5---sn-f5f7lne6.googlevideo.com)|172.217.133.74|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://r5---sn-4g5e6nss.googlevideo.com/videoplayback?sparams=dur,ei,expire,gcr,id,initcwndbps,ip,ipbits,itag,lmt,mime,mm,mn,ms,mv,pl,ratebypass,requiressl,source&signature=4C086E762A589DF1BFAECAD77A91CCB06E321CCD.13CDF79D8AC8D6C915D9CE561F5E0BFBF8751A9B&itag=22&c=WEB&id=o-AA-YvKNVrqlsUju19HHHyW_YI6osduGtVnZoB0QZj0eQ&ip=&ei=ZQLMWpTxB7ym7ATgw4-wCw&pl=13&ipbits=0&dur=3130.142&fvip=5&source=youtube&lmt=1510089918251993&ratebypass=yes&expire=1523340997&key=cms1&mime=video%2Fmp4&gcr=pl&requiressl=yes&redirect_counter=1&cm2rm=sn-f5fy76&req_id=35a7a6728ceda3ee&cms_redirect=yes&mm=34&mn=sn-4g5e6nss&ms=ltu&mt=1523319336&mv=m [following]
--2018-04-10 03:17:08--  https://r5---sn-4g5e6nss.googlevideo.com/videoplayback?sparams=dur,ei,expire,gcr,id,initcwndbps,ip,ipbits,itag,lmt,mime,mm,mn,ms,mv,pl,ratebypass,requiressl,source&signature=4C086E762A589DF1BFAECAD77A91CCB06E321CCD.13CDF79D8AC8D6C915D9CE561F5E0BFBF8751A9B&itag=22&c=WEB&id=o-AA-YvKNVrqlsUju19HHHyW_YI6osduGtVnZoB0QZj0eQ&ip=&ei=ZQLMWpTxB7ym7ATgw4-wCw&pl=13&ipbits=0&dur=3130.142&fvip=5&source=youtube&lmt=1510089918251993&ratebypass=yes&expire=1523340997&key=cms1&mime=video%2Fmp4&gcr=pl&requiressl=yes&redirect_counter=1&cm2rm=sn-f5fy76&req_id=35a7a6728ceda3ee&cms_redirect=yes&mm=34&mn=sn-4g5e6nss&ms=ltu&mt=1523319336&mv=m
Resolving r5---sn-4g5e6nss.googlevideo.com (r5---sn-4g5e6nss.googlevideo.com)... 173.194.182.202
Connecting to r5---sn-4g5e6nss.googlevideo.com (r5---sn-4g5e6nss.googlevideo.com)|173.194.182.202|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 316120049 (301M) [video/mp4]
Saving to: 'videoplayback@sparams=dur%2Cei%2Cgcr%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cexpire&signature=9A40973E10581CBAFEC407C289DCC9D2ADD43AF2.A729230062EA5A24F0CA1565FD3A0'

videoplayback@sparams=dur%2Ce   0%[                                                  ]   1.33M  6.20MB/s    in 0.2s

2018-04-10 03:17:09 (6.20 MB/s) - Connection closed at byte 1399622. Retrying.
```

partially downloaded 1.3MB file has proper headers of h264+mp4 audio clip, even size of 300MB is valid considering

> 140          m4a        audio only DASH audio  130k , m4a_dash container, mp4a.40.2@128k, 47.41MiB
> 136          mp4        1280x720   720p 2266k , avc1.4d401f, 25fps, video only, 254.23MiB


so file is there on the server, google just doesnt want us to have it. 
Same as my issue #16140
